packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -6,7 +6,7 @@
                 {{ $t('labels.addressGroup') }}
             </legend>
             <form-field
-                :value="fulfilment.address.line1"
```

Comment:
@oliversweb and I thought that the nesting here was unnecessary, as the fulfilment address will be the only address on the page.